Cargo.toml

```
@@ -12,10 +12,10 @@ indexmap = { version = "1.9.2", features = ["serde"] }
 once_cell = { version = "1.16.0" }
 serde = { version = "1.0.130", features = ["derive", "rc"] }
 serde_json = { version = "1.0.81" }
-starknet-crypto = { version = "0.2.0" }
+starknet-crypto = { git = "https://github.com/xJonathanLEI/starknet-rs", rev = "5d3f9d7cc6df0d01a299f0fb0ebc00ec2f28b983" }
```

```suggestion
starknet-crypto = { version = "0.4.3" }
```

Hey somehow came across this PR. This commit you're referencing here was actually published on crates.io as `0.4.3`. So there's no need to resort to installing from Git here :)